crates/hive-utils/src/hive/genesis.rs

```diff
@@ -107,6 +112,8 @@ pub async fn serialize_hive_to_madara_genesis_config(
         ("externally_owned_account", eoa_class_hash),
         ("account_proxy_class_hash", account_proxy_class_hash),
         ("blockhash_registry_address", *BLOCKHASH_REGISTRY_ADDRESS),
+        // TODO: Use DEPLOY_FEE constant https://github.com/kkrt-labs/kakarot-rpc/pull/431/files#diff-88f745498d0aaf0b185085d99a74f0feaf253f047babc85770847931e7f726c3R125
+        ("deploy_fee", FieldElement::from(100000_u64)),
```

Comment:
@bajpai244 will this become a problem? since it's not supposed to be a constant

Reply:
So, this value can always be updated later on, u have to firstly initialize Kakarot with one `deploy_fee` which seems to be the case here, and then an off-chain service can keep updating this value, so technically we are not making this a constant here, we are just initializing Kakarot.